grafana.yaml

```diff
@@ -0,0 +1,79 @@
+package:
+  name: grafana
+  version: 10.2.0
+  epoch: 0
+  description: The open and composable observability and data visualization platform.
+  copyright:
+    - license: AGPL-3.0-or-later
+  dependencies:
+    runtime:
+      - bash
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
```

Comment:
nit: Generally `ca-certificates-bundle` is included in `build-base --> wolfi-baselayout.`